audio-worklet/lib/param-controller.js

```diff
@@ -16,44 +16,54 @@
 class ParamController {
   /**
    * Event handler and layout for audio parameters with numeric input values
-   * @param  {String} parentId id for parent element
-   * @param  {Function} onChangeCallback callback to trigger on input
-   * @param  {String} options.name the name of the parameter
-   * @param  {String} options.type the type of input (only support for range)
-   * @param  {String} options.min the minimum possible value
-   * @param  {String} options.max the maximum possible value
-   * @param  {String} options.step the parameter's increment value
-   * @param  {String} options.default the default value of the parameter
+   * @param {String} parentId id for parent element.
+   * @param {Function} onChangeCallback Callback to trigger on input.
+   * @param {Object} options
+   * @param {String} options.name The name of the parameter.
+   * @param {String} options.id A variable name associated with the parameter.
+   * @param {String} options.type The type of input (only support for range).
+   * @param {Number} options.min The minimum possible value.
+   * @param {Number} options.max The maximum possible value.
+   * @param {Number} options.step The parameter's increment value.
+   * @param {Number} options.default The default value of the parameter.
    */
   constructor(parentId, onChangeCallback, options) {
     if (options == null) options = {};
+    
+    this.controller_ = document.createElement('input');
+    this.name_ = options.name || 'Parameter';
+    this.id_ = options.id || this.name_;
+    this.controller_.type = options.type || 'range';
+    this.controller_.min = options.min || 0;
+
+    if (options.max == null)
```

Comment:
Use `===` whenever you can. Also what you want to compare with is `undefined`, not `null`.